cg/models/pipeline_qc_metrics/pipeline_qc_metrics.py

```diff
@@ -0,0 +1,113 @@
+from datetime import datetime
+
+from pydantic import BaseModel
+
+
+class LaneMetric(BaseModel):
+    """Model for lane specific sequencing metrics."""
+
+    lane: int
+    number_of_reads: int
+    _yield: int | None = None
+
+
+class ConcatenatedLaneMetric(BaseModel):
+    """Model for concatenated lane sequencing metrics."""
+
```

Comment:
Yes, ideally this will become onle "LaneMetric" in the future when all pipelines do lane-wise alignment. But for now we have to support one or the other